test/DAREditor/JSONDescrepency.cs

```diff
@@ -0,0 +1,25 @@
+﻿// // Copyright (c) Microsoft. All rights reserved.
+// // Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Newtonsoft.Json;
+using OpenDebug;
+namespace DAREditor
+{
+    class JSONDescrepency
```

Comment:
>JSONDescrepency [](http://example.com/codeflow?start=10&length=15)

I don't understand the name of this class. Did you mean something like `JsonHandler`?